.github/actions/shared-utils/fetch-retry.js

```diff
@@ -16,12 +16,16 @@ export async function fetchRetry (url, options) {
         retry: undefined
       })
 
-      if (result.ok) {
+      status = result?.status, 
```

Comment:
This actually doesn't make sense to have in a while loop any more since it is a recursive fn call.